applications/ContactStructuralMechanicsApplication/tests/cpp_tests/test_mixedulm_linear_solver.cpp

```diff
@@ -662,7 +660,7 @@ namespace Kratos
                 for (std::size_t j = 0; j < system_size; ++j) {
                     if ((((i == 0 || i == 1 || i == 2) && ((j == system_size - 1) || (j == system_size - 2) || (j == system_size - 3))) || ((j == 0 || j==1 || j==2) && ((i == system_size - 1) || (i == system_size - 2) || (i == system_size - 3))) || (i == 6 && (j == 10 || j == 11)) || (i == 7 && (j == 9 || j == 11)) || (i == 8 && (j == 9 || j == 10))) == false) {
                         count += 1.0;
-                        A.push_back(i, j, std::sqrt(count));
+                        A.push_back(i, j, 5.0 + std::sqrt(count));
```

Comment:
I do this to create a "random" system better conditioned (that's why I used AMGCL)